src/load-data.js

```diff
@@ -1,5 +1,33 @@
 'use strict'
 
-export default function loadDataIntoLocalStorage ({ serializedData = {} }) {
-  throw new Error('Not implemented')
+const TILL_KEY = 'SPLITIO.splits.till'
+
+export default function loadDataIntoLocalStorage ({ serializedData = {} }, windowLocalStorage = {}) {
+  const { segmentsData, since, splitsData, userId, usingSegmentsCount } = serializedData
```

Comment:
where did these come from? the BE won't know user ID

Reply:
These are the parameters based on what we want to load into localStorage (from the google doc).  
`userId` would come from partners.  
My understanding is that the partner would be passing this into the `hivemind-javascript-sdk`, similar to how they pass it into the `hivemind-node-sdk`.

Reply:
`userId` won't be returned with the rest of the serialized data though

Reply:
Discussed offline.
`userId` is not a part of the serialized data. It is a separate parameter that we will be getting from partners, not the backend.